apis/groups/v1alpha1/group_types.go

```diff
@@ -113,7 +113,7 @@ type GroupParameters struct {
 
 	// Disable email notifications.
 	// +optional
-	EmailsDisabled *bool `json:"emailsDisabled,omitempty"`
+	EmailsEnabled *bool `json:"emailsEnabled,omitempty"`
```

Comment:
To ensure backwards compatibility we should keep `EmailsDisabled` as a deprecated property and add `EmailsEnabled` as the new recommended option.

Similar to what is described in https://github.com/crossplane-contrib/provider-gitlab/issues/141#issuecomment-2247099937 the field `EmailsEnabled` should be late-initialized during observe (pseudocode):

```go
cr.Spec.ForProvider.EmailsEnabled = LateInitialize(cr.Spec.ForProvider.EmailsEnabled, cr.Spec.ForProvider.EmailsDisabled)
````

And after that `EmailsEnabled` can be used throughout the rest of the controller.